src/rinterface.c

```diff
@@ -7951,7 +7951,7 @@ SEXP R_igraph_hrg_fit(SEXP graph, SEXP hrg, SEXP start, SEXP steps) {
                                         /* Convert input */
   R_SEXP_to_igraph(graph, &c_graph);
   if (0 != R_SEXP_to_hrg_copy(hrg, &c_hrg)) {
-    igraph_error("", __FILE__, __LINE__, IGRAPH_ENOMEM);
+    igraph_error("internal error in R_SEXP_to_hrg_copy()", __FILE__, __LINE__, IGRAPH_ENOMEM);
```

Comment:
```suggestion
    igraph_error("Insufficient memory to create HRG object.", __FILE__, __LINE__, IGRAPH_ENOMEM);
```

Please don't use the phrasing "internal error" for errors that are expected, such as out-of-memory.